testApp/src/CurrentUserStatusTransactions.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { Didomi } from '@didomi/react-native';
+import Getter from './GetterCall';
+
+// UI element used to contain buttons related to tests that involve `CurrentUserStatusTransaction`.
+export default function CurrentUserStatusTransactions() {
```

Comment:
Could be renamed for example `CurrentUserStatusTransactionCall`, to avoid confusion

Reply:
currently the `Call` prefix is only used for individual components e.g.: `GetterCall`, `SetterCall` so it might be confusing to call this one `CurrentUserStatusTransactionsCall` since this component represents a list. Equivalent components end in plural e.g.: `Getters`, `Setters` so I think `CurrentUserStatusTransactions` is still consistent. An alternative could be `CurrentUserStatusTransactionsList`.